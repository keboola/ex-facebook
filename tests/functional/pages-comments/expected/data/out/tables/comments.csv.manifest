{"primary_key":["id","parent_id"],"columns":["id","ex_account_id","fb_graph_node","parent_id","message","created_time","from_name","from_id"]}