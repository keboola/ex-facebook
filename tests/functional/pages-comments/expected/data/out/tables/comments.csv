"122093300126070033_849942843423502","133353749860945","page_feed_comments","133353749860945_122093300126070033","second comment","2023-09-28T07:58:13+0000","Test extractor page","133353749860945"
"122093300126070033_336066748802768","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","dasdasd","2023-09-28T08:06:00+0000","Test extractor page","133353749860945"
"122093300126070033_840871214190749","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","asdasd","2023-09-28T08:06:02+0000","Test extractor page","133353749860945"
"122093300126070033_333035919251775","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","dqwdqwqdw","2023-09-28T08:06:05+0000","Test extractor page","133353749860945"
"122093300126070033_783946326752674","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","wqdqwdqw","2023-09-28T08:06:07+0000","Test extractor page","133353749860945"
"122093300126070033_337285515479278","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","wqdwqdqwd","2023-09-28T08:06:08+0000","Test extractor page","133353749860945"
"122093300126070033_345054541214693","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","fg","2023-09-28T08:06:09+0000","Test extractor page","133353749860945"
"122093300126070033_658716936354239","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","ewg","2023-09-28T08:06:09+0000","Test extractor page","133353749860945"
"122093300126070033_658909099548530","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qefwe","2023-09-28T08:06:08+0000","Test extractor page","133353749860945"
"122093300126070033_874399957629134","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","few","2023-09-28T08:06:08+0000","Test extractor page","133353749860945"
"122093300126070033_1697394000723430","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","fgew","2023-09-28T08:06:08+0000","Test extractor page","133353749860945"
"122093300126070033_309011961745128","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","wef","2023-09-28T08:06:09+0000","Test extractor page","133353749860945"
"122093300126070033_329927309515597","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","wef...další zanoření","2023-09-28T08:06:09+0000","Test extractor page","133353749860945"
"122093300126070033_802262648362545","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","dqw","2023-09-28T08:06:10+0000","Test extractor page","133353749860945"
"122093300126070033_876321904008991","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","w","2023-09-28T08:06:09+0000","Test extractor page","133353749860945"
"122093300126070033_1004450097496957","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","dq","2023-09-28T08:06:10+0000","Test extractor page","133353749860945"
"122093300126070033_1531306704373532","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","weg","2023-09-28T08:06:09+0000","Test extractor page","133353749860945"
"122093300126070033_825017749071778","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","wd","2023-09-28T08:06:10+0000","Test extractor page","133353749860945"
"122093300126070033_849865883206219","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwdf","2023-09-28T08:06:10+0000","Test extractor page","133353749860945"
"122093300126070033_1725649441196610","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwf","2023-09-28T08:06:10+0000","Test extractor page","133353749860945"
"122093300126070033_181803661549536","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","f","2023-09-28T08:06:11+0000","Test extractor page","133353749860945"
"122093300126070033_247601224946486","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwf","2023-09-28T08:06:11+0000","Test extractor page","133353749860945"
"122093300126070033_337571428737446","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","fwq","2023-09-28T08:06:11+0000","Test extractor page","133353749860945"
"122093300126070033_1006441300623002","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qw","2023-09-28T08:06:11+0000","Test extractor page","133353749860945"
"122093300126070033_1719360465166701","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwf","2023-09-28T08:06:11+0000","Test extractor page","133353749860945"
"122093300126070033_988240375784612","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwfqwfqw","2023-09-28T08:06:16+0000","Test extractor page","133353749860945"
"122093300126070033_247190077865979","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","f","2023-09-28T08:06:17+0000","Test extractor page","133353749860945"
"122093300126070033_299374059468741","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","wq","2023-09-28T08:06:17+0000","Test extractor page","133353749860945"
"122093300126070033_629684322613574","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwf","2023-09-28T08:06:17+0000","Test extractor page","133353749860945"
"122093300126070033_1025486308500545","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","fqw","2023-09-28T08:06:17+0000","Test extractor page","133353749860945"
"122093300126070033_1028100411964921","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwf","2023-09-28T08:06:17+0000","Test extractor page","133353749860945"
"122093300126070033_1385996901990669","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","fqw","2023-09-28T08:06:17+0000","Test extractor page","133353749860945"
"122093300126070033_696817588978298","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","fqw","2023-09-28T08:06:18+0000","Test extractor page","133353749860945"
"122093300126070033_830973568823830","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","qwf","2023-09-28T08:06:18+0000","Test extractor page","133353749860945"
"122093300126070033_870703694085243","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","wq","2023-09-28T08:06:18+0000","Test extractor page","133353749860945"
"122093300126070033_1399619967639909","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","f","2023-09-28T08:06:18+0000","Test extractor page","133353749860945"
"122093300126070033_839092987596784","133353749860945","page_feed_comments_comments","122093300126070033_849942843423502","fq","2023-09-28T08:06:19+0000","Test extractor page","133353749860945"
"122093300126070033_635586898738911","133353749860945","page_feed_comments","133353749860945_122093300126070033","first comment","2023-09-28T07:54:26+0000","Test extractor page","133353749860945"
